src/charts/scatter-plot.js

```diff
@@ -698,9 +698,30 @@ define(function(require) {
             let { closestPoint } = getPointProps(e);
             let d = getPointData(closestPoint);
 
+            handleClickAnimation(d);
+
             dispatcher.call('customClick', e, d, d3Selection.mouse(e), [chartWidth, chartHeight]);
         }
 
+        /**
+         * Applies animation on data point click
+         * @param {Object} dataPoint
+         * @return {void}
+         * @private
+         */
+        function handleClickAnimation(dataPoint) {
+            highlightCircle
+                .transition()
+                .ease(ease)
```

Comment:
Nit: I usually indent everything after the .transition, to mark what is going to happen